package.json

```diff
@@ -87,7 +87,7 @@
     "tsd": "^0.7.4"
   },
   "peerDependencies": {
-    "@hapi/hapi": "^19.0.0",
+    "@hapi/hapi": "^20.0.0",
```

Comment:
Should we do this as `"@hapi/hapi": "^19 || ^20"` since it still works fine with @hapi v19.